scripts/pipelines/azure-devops/templates/aks/obtain-dns.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+ip="$(kubectl get svc nginx-ingress-nginx-ingress-controller --namespace nginx-ingress --kubeconfig "$1"/kubeconfig -o jsonpath='{.status.loadBalancer.ingress[0].ip}')"
```

Comment:
```suggestion
ip="$(kubectl get svc nginx-ingress-nginx-ingress-controller --namespace nginx-ingress --kubeconfig "$1/kubeconfig" -o jsonpath='{.status.loadBalancer.ingress[0].ip}')"
```

Although better pass KUBECONFIG env variable.